AmplifyPlugins/Core/AWSPluginsCore/Sync/MutationSync/MutationSyncMetadata.swift

```diff
@@ -8,9 +8,31 @@
 import Amplify
 
 public struct MutationSyncMetadata: Model {
-
     public let id: Model.Identifier
     public var deleted: Bool
     public var lastChangedAt: Int
     public var version: Int
+    
+    public var modelId: String {
+        id.components(separatedBy: "|").last ?? ""
+    }
+    public var modelName: String {
+        id.components(separatedBy: "|").first ?? ""
+    }
+    
+    public init(id: Model.Identifier, deleted: Bool, lastChangedAt: Int, version: Int) {
```

Comment:
is `id` a type of `Model.Identifier`? Instead should we call it `String` or better create a MutationSyncMetadata.Identifier?